test/components/fetchers/test_link_content_fetcher.py

```diff
@@ -166,19 +183,6 @@ def test_link_content_fetcher_text(self):
         assert "url" in first_stream.meta and first_stream.meta["url"] == TEXT_URL
         assert first_stream.mime_type == "text/plain"
 
-    @pytest.mark.integration
-    def test_link_content_fetcher_pdf(self):
```

Comment:
we already perform HTTP requests for several different file types in other tests, so this is not needed